website/photos/models.py

```diff
@@ -98,6 +115,267 @@ class Meta:
         unique_together = ["photo", "member"]
 
 
+# class PersonOnPhoto(models.Model):
+#     photo = models.ForeignKey(
+#         Photo,
+#         verbose_name=_("Photo"),
+#         null=False,
+#         blank=False,
+#         related_name="persons",
+#         on_delete=models.CASCADE,
+#     )
+#     member = models.ForeignKey(
+#         Member,
+#         verbose_name=_("Member"),
+#         null=True,
+#         blank=False,
+#         on_delete=models.SET_NULL,
+#     )
+#     self_indicated = models.BooleanField(
+#         verbose_name=_("Self-indicated"), default=False
+#     )
+#
+#     def __str__(self):
+#         return str(self.member) + " " + _("is on") + " " + str(self.photo)
+#
+#     class Meta:
+#         unique_together = ["photo", "member"]
+
+
+class FaceEncoding(models.Model):
```

Comment:
Maybe we should put the different models in subfiles here now too

Reply:
Im actually thinking about moving the whole thing to a separate app